src/clj/rems/api/schema.clj

```diff
@@ -129,7 +129,7 @@
 ;;; template for a form field, before answering
 (s/defschema FieldTemplate
   {:field/id schema-base/FieldId
-   :field/type (s/enum :attachment :date :description :email :header :label :multiselect :option :text :texta :table)
+   :field/type (s/enum :attachment :date :description :email :header :label :multiselect :option :phonenumber :text :texta :table)
```

Comment:
Minor nitpick: Would `:phone` be more in line with `:email`? If you want to call it a phone number, I think `:phone-number` would be more consistent with our api in general.

Reply:
I can fix that, no big deal